pandas-stubs/core/frame.pyi

```diff
@@ -1011,7 +1011,9 @@ class DataFrame(NDFrame, OpsMixin):
     def stack(
         self, level: Level | list[Level] = ..., dropna: _bool = ...
     ) -> DataFrame | Series[Any]: ...
-    def explode(self, column: _str | tuple, ignore_index: _bool = ...) -> DataFrame: ...
+    def explode(
+        self, column: _str | tuple | Sequence[Hashable], ignore_index: _bool = ...
```

Comment:
You can just use `Sequence[Hashable]` by itself, as that will match `_str` and `tuple` as well.